dask/dataframe/groupby.py

```diff
@@ -895,8 +895,15 @@ def _groupby_apply_funcs(df, *index, **kwargs):
 
 
 def _compute_sum_of_squares(grouped, column):
-    base = grouped[column] if column is not None else grouped
-    return base.apply(lambda x: (x ** 2).sum())
+    # Note: CuDF cannot use `groupby.apply`.
```

Comment:
Just a clarification, we can use `groupby.apply` but we should avoid it for both Pandas and cuDF whenever possible as it goes through a slower iteration based code path.